lib/whitehall/asset_manager_storage.rb

```diff
@@ -1,38 +1,16 @@
-class Whitehall::AssetManagerStorage < CarrierWave::Storage::Abstract
-  def store!(file)
-    original_file = file.to_file
-    temporary_location = ::File.join(
-      Whitehall.asset_manager_tmp_dir,
-      SecureRandom.uuid,
-      ::File.basename(original_file),
-    )
-
-    FileUtils.mkdir_p(::File.dirname(temporary_location))
-    FileUtils.cp(original_file, temporary_location)
-    draft = uploader.assets_protected?
-
-    if uploader.model.respond_to?(:attachable)
-      attachable_model_class = uploader.model.attachable && uploader.model.attachable.class.to_s
-      attachable_model_id = uploader.model.attachable && uploader.model.attachable.id
+class Whitehall::AssetManagerStorage
+  class TempStorage
+    def self.store!(file)
+      temporary_location = ::File.join(
+        Whitehall.asset_manager_tmp_dir,
+        SecureRandom.uuid,
+        ::File.basename(file),
+      )
+
+      FileUtils.mkdir_p(::File.dirname(temporary_location))
+      FileUtils.cp(file, temporary_location)
+      temporary_location
```

Comment:
I considered adding a unit test for this, but given it interacts so extensively with the file system and it's covered by integration and end to end tests I decided not to